app/controllers/embedded_terraform_repository_controller.rb

```diff
@@ -0,0 +1,165 @@
+      javascript_redirect(:action => 'edit', :id => id)
+    when "embedded_configuration_script_source_add"
+      javascript_redirect(:action => 'new')
+    when "embedded_terraform_repositories_reload"
+      show_list
+      render :update do |page|
+        page << javascript_prologue
+        page.replace("gtl_div", :partial => "layouts/gtl")
+      end
+    when "embedded_terraform_repository_reload"
+      show
+      render :update do |page|
+        page << javascript_prologue
+        page.replace("main_div", :template => "embedded_terraform_repository/show")
+      end
+    when "ansible_repository_tag"
+      tag(self.class.model)
+    when "embedded_configuration_script_payload_tag" # templates from nested list
+      tag(ManageIQ::Providers::EmbeddedTerraform::AutomationManager::Template)
+    end
+  end
+
+  def edit
+    assert_privileges('embedded_configuration_script_source_edit')
+    @record = self.class.model.find(params[:id])
+    drop_breadcrumb(:name => _("Edit a Repository \"%{name}\"") % {:name => @record.name},
+                    :url  => "/embedded_terraform_repository/edit/#{@record.id}")
+    @title = _("Edit Repository \"%{name}\"") % {:name => @record.name}
+    @id = @record.id
+    @in_a_form = true
+  end
+
+  def new
+    assert_privileges('embedded_configuration_script_source_add')
+    drop_breadcrumb(:name => _("Add a new Repository"), :url => "embedded_terraform_repository/new")
+    @title = _("Add new Repository")
+    @id = 'new'
+    @in_a_form = true
+  end
+
+  def check_button_rbac
+    # Allow reload to skip RBAC check
+    if %w[embedded_terraform_repository_reload embedded_terraform_repositories_reload].include?(params[:pressed])
+      true
+    else
+      super
+    end
+  end
+
+  def show_list
+    assert_privileges('embedded_configuration_script_source_view')
+    super
+  end
+
+  def show
+    assert_privileges('embedded_configuration_script_source_view')
+    super
+  end
+
+  def show_output
+    drop_breadcrumb(:name => _("Refresh output"), :url => show_output_link)
+    @showtype = 'output'
+  end
+
+  def display_templates
+    nested_list(ManageIQ::Providers::EmbeddedTerraform::AutomationManager::Template, :breadcrumb_title => _('Templates'))
+  end
+
+  def repository_refresh
+    assert_privileges("embedded_configuration_script_source_refresh")
+    checked = find_checked_items
+    checked[0] = params[:id] if checked.blank? && params[:id]
+
+    self.class.model.where(:id => checked).each do |repo|
+      repo.sync_queue
+      add_flash(_("Refresh of Repository \"%{name}\" was successfully initiated.") % {:name => repo.name})
+    rescue StandardError => ex
+      add_flash(_("Unable to refresh Repository \"%{name}\": %{details}") % {:name    => repo.name,
+                                                                             :details => ex},
+                :error)
+    end
+
+    javascript_flash
+  end
+
+  def toolbar
+    require 'byebug'
+    byebug
```

Comment:
👋 🐛

Reply:
Hey @GilbertCherrie can you amend the commit that added this rather than adding a commit that removes it? https://github.com/ManageIQ/manageiq-providers-embedded_terraform/pull/3/commits/048e82c80281e3050dafe745afcdfbf3e2f70b97

Just so we don't have this in the git history